### DISCUSSION TITLE
Team Links (or shared vs. private folders)

### DISCUSSION THREAD

**Discussion by ReessKennedy** [shlinkio/shlink#2235] [Ideas]
My research indicates it's not but just asking, is it possible to: 
- have multiple unique logins to one shlink server
- allow each user to have his/her own private links
- alow each user to create shared tags or folder or domains of links to share with other logins

Shlink is such a cool and useful tool and I can see this being really useful, especially for marketing teams that use this when you want to have some stuff you're doing privately and some stuff you do collaboratively and want to share the click stats for those collaborative things.

**Comment by acelaya** [comment]
Unfortunately that's not possible at the moment, but there's some work in progress to have a new shlink-dashboard that will provide some of those capabilities. 

* Here you can find some context https://github.com/shlinkio/shlink-web-client/issues/338
* And this is the repository https://github.com/shlinkio/shlink-dashboard
* And finally, you can find some details of Shlink architecture here https://shlink.io/documentation/advanced/shlink-architecture/

**Reply by ReessKennedy** [reply]
Wow! Cool!
